documentation/index.ts

```diff
@@ -0,0 +1,497 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import j2s, { ComponentsSchema } from 'joi-to-swagger';
+import { Schema } from 'joi';
+import path from 'path';
```

Comment:
I get a typescript error here:

TS1259: Module '"path"' can only be default-imported using the 'esModuleInterop' flag  path.d.ts(167, 5): This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.

Reply:
I don't have one anymore. You?

Reply:
Yes, fixed